scripts/tiff_unpacker.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (C) 2023 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+"""
+This is a Utility script to unpack tiff files into a directory of images using lossless compression.
+This script is temporary while larger work is undertaken to improve how datasets are stored and accessed within Mantid.
+
+Usage:
+    python tiff_unpacker.py <path to tiff file> <path to output directory>
+    python tiff_unpacker.py
+        -s <path to tiff file>
+        -d <path to output directory>
+        -r <start> <stop>
+        -n <naming convention>
+"""
+import argparse
+from tqdm import tqdm
+from pathlib import Path
+from tifffile import TiffFile, imwrite
+from logging import getLogger, basicConfig, INFO
+
+basicConfig(format='%(asctime)s %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p', level=INFO)
+logger = getLogger(__name__)
+
+
+def unpack_tiff(source: Path, destination: Path, img_range: tuple[int, int], naming_convention: str) -> None:
+    """
+    Unpack a tiff file into a directory of LZW compressed tif images
+
+    Parameters:
+        source: Path to tiff file
+        destination: Path to output directory
+
+    Returns:
+        None
+    """
+    with TiffFile(source) as tif:
+        for i, img in enumerate(tqdm(tif.pages[img_range[0]:img_range[1]], desc="Unpacking tiff file")):
+            imwrite(destination / f"{naming_convention}{i}.tiff",
+                    img.asarray(),
+                    compression="zlib",
+                    predictor=True,
+                    metadata={
+                        'axes': 'TZCYX',
+                        'unit': 'um'
+                    })
```

Comment:
I think this is claiming that we are writing 5D data, but we should have 2D "YX" data here. We could use the axis from the page.
`'axes': img.axes`
I'm not sure where the `unit` is coming from